config/environments/development.rb

```diff
@@ -29,6 +29,10 @@
     config.cache_store = :null_store
   end
 
+  # Disable request forgery protection in dev environment.
```

Comment:
I'm a little confused by this (and, I guess, I worry it isn't just limited to local development); it seems like it might be worth looking into a little more to see if there's a better way to handle this without globally disabling such an important security feature.

Reply:
I added this to the controller instead:

`skip_before_action :verify_authenticity_token, only: :create, unless: :production?`

Reply:
Moved this to PaymentController